tests/utilities.lisp

```diff
@@ -24,6 +24,34 @@
                 :else
                   :collect form)))))
 
+(defmacro with-execution-modes ((&rest modes) &body body)
+  "Execute body in a variety of execution modes. MODES can be:
+
+    - :INTERPRET, where any calls to QVM:RUN will execute immediately.
```

Comment:
Can you include a desc of when a particular mode should be used

Reply:
done